plugins/gradle/example-client/build.gradle

```diff
@@ -8,11 +8,13 @@ plugins {
     // use the latest available version:
     // https://plugins.gradle.org/plugin/io.github.kobylynskyi.graphql.codegen
     id "io.github.kobylynskyi.graphql.codegen" version "4.0.2-SNAPSHOT"
+    id "org.jetbrains.kotlin.jvm" version "1.3.71"
 }
 
 mainClassName = "io.github.kobylynskyi.order.Application"
 
 dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
```

Comment:
I have thought about it, but I think the example of gradle is too heavy. Unlike Scala example, I find that it uses spring and mongodb.So I can add the same (simple)example as SBT. This example only refers to the plug-in and the library itself. What do you think?

Reply:
People might be confused by the kotlin plugin and stdlib dependency here if they just want to use the lib for Java codegen. I also believe it would be much better to separate. I know it's a bit heavy but it would be worth it.
If you don't want to do it I think it would be less confusing and maybe better to have no Kotlin example at all rather than mix it with the Java one.

Reply:
OK, I'll take the time to write an example and try the generated code.